reconcile/test/test_openshift_resource.py

```diff
@@ -132,3 +139,61 @@ def test_has_owner_reference_false():
         }
         openshift_resource = OR(resource, TEST_INT, TEST_INT_VER)
         assert not openshift_resource.has_owner_reference()
+
+    @staticmethod
+    def test_check_temp_role_after_expiration_date():
+        expiration_date = date.today() + \
+                timedelta(days=(openshift_rolebindings.EXPIRATION_MAX+1))
+        resource = mock_openshift_role_bindings(expiration_date)
+        for r in resource:
+            assert openshift_rolebindings.role_still_valid(r) is False
+
+    @staticmethod
+    def test_check_temp_role_before_expiration_date():
+        expiration_date = date.today() + \
+                timedelta(days=(openshift_rolebindings.EXPIRATION_MAX-1))
+        resource = mock_openshift_role_bindings(expiration_date)
+        for r in resource:
+            assert openshift_rolebindings.role_still_valid(r) is True
+
+    @staticmethod
+    def test_check_temp_role_no_expiration_date():
+        resource = mock_openshift_role_bindings_no_expiration_date()
+        for r in resource:
+            assert openshift_rolebindings \
+                .has_valid_expiration_date(r) is True
```

Comment:
Easier:
```suggestion
            assert openshift_rolebindings \
                .has_valid_expiration_date(r)
```
(maybe this fits in one line now)